src/zenml/integrations/mlflow/mlflow_environment.py

```diff
@@ -98,3 +107,107 @@ def deactivate(self) -> None:
     def tracking_uri(self) -> str:
         """Returns the MLflow tracking URI for the current stack."""
         return self._mlflow_tracking_uri
+
+
+class MLFlowStepEnvironment(BaseEnvironmentComponent):
+    """Provides information about an MLflow step environment.
+    To access it inside your step function:
```

Comment:
As an aside / comment, I really like these code-inside-docstrings examples. I think we should do it throughout our codebase where possible.